spec/features/user_enters_valid_email_spec.rb

```diff
@@ -1,5 +1,39 @@
 require 'rails_helper'
 
-RSpec.feature 'Sign-up email', type: :feature do
-  
+RSpec.feature 'Sign-in email', type: :feature do
+  scenario "user enter blank password with valid e-mail" do
+    sign_up_with_sign_out
+    fill_in('Email', :with => 'user@gmail.com')
+    click_button "Log in"
```

Comment:
Could add `expect(current_path)` expectation to make sure user is on the page you expect them to be on